src/test/java/com/google/api/codegen/testdata/py/python_library.baseline

```diff
@@ -4175,14 +4172,11 @@ names = [names_element]
 shelves_element = 'Classics'
 shelves = [shelves_element]
 
-# Iterate over results one page at a time
+# Iterate over all results
```

Comment:
I would keep this as "one page at a time", since we are showing how to do that (and could insert code to print a page header).